### DISCUSSION TITLE
Stellarium closes when I'm with my EQ8 telescope using GSServer

### DISCUSSION THREAD

**Discussion by Visssio** [Stellarium/stellarium#2842] [Q&A]
Hello
With these last versions using stellarium, I can conect my telescope, I see it goes a little slow, but it´s ok

When I go to one object (planet, star ...), the telescope moves and no problem but when I go to another program or window (firecapture for example) usually Stellarium disconect and I have to go again to Stellarium, connect the telescope, move again to the last position ... and once again, in a moment, stellarium disconect  ... 

This situation is a little worried because I have to spend a lot of time to do anything

What can I do????

I use GSServer with my EQ8 mount

Thanks a lot

**Comment by gzotti** [comment]
We have a few other reports about recent Stellarium versions crashing silently when switching between programs with telescope connected. (ASCOM?). Does Stellarium survive on your PC and just loses connection, or do you have to restart Stellarium as well? 

I don't know what GSServer is, sorry. We are in real need of developers with such mounts who could fix ASCOM problems (and are driven by their own motivation to do so). My EQ3 always worked with built-in protocols. (But I don't find even time to use it now...)

**Reply by Visssio** [reply]
Thanks for response

GS Server is Green Swamp Server (https://greenswamp.org)
And right, I am using ASCOM

Abot this problem, I have to restart stellarium all the time 
As I said, my mount is EQ8 ... maybe is a bug, 
I am new on this but I would say this is a recent problem because I didnt have this a few month ago and don´t know what can I do

Thanks

**Reply by gzotti** [reply]
OK. Yes, this has been reported, but due to lack of such instruments (the actual hardware) no developer can further diagnose it. Somebody with such instrument could help us by doing it and sending a solution. If an older version like 0.21 works for you, maybe revert on the computer that drives your telescope. You can also install another version (into a different directory!) and use this for guiding.

**Comment by oleksandrahaidai** [comment]
hi! had the same problem... and still had no solution

**Reply by vzr314** [reply]
I also have problem with branch 1.x Can't work 3 minutes without crashing. I am using different telescopes via ASCOM (OnStep wifi/cable, EQMOD, GSServer) it is crashing like crazy. Now I am reverting back to 0.22.2 to check if it is stable as I remember...

**Reply by gzotti** [reply]
"crashing like crazy" reads like "every 5 seconds also during idle operation". How can we ever reproduce that?
Please tell us what interaction on your side causes the crash. Is it "Whatever you do", or is it one specific action?
https://github.com/Stellarium/stellarium/wiki/FAQ#Reporting_a_crash

**Reply by vzr314** [reply]
It meansn I minimize window and do other things, for example work in NINA, it crashes meanawhile or when I try restore it. Or if I just move and zoom the sky in Stellarium in random directions kust a little bit faster, the OS displays "not respinding" and close. (Stellarium x64 1.1 on Windows 11, 11th gen i7 CPU and Intel Iris Xe GPU). And yeah, I have shitload of plugins (remote control, telescope control, angle measure, and all star catalogues installed).

**Comment by alex-w** [comment]
Please check qt5-based package

**Reply by buckeyestargazer** [reply]
I have the same issue.  Stellarium will just disappear and I haven't been able to determine if there is a trigger for it.  I'm using telescope control with the Astro-Physics ASCOM driver.

**Reply by vzr314** [reply]
Okay, I think Qt5 release is working normally from what I can tell after few rounds of testing. I don't know if this is because of Qt5 or these bugs are fixes in most recent release.
